modules/efk-logging-elasticsearch-status-viewing.adoc

```diff
@@ -128,3 +131,31 @@ This status message indicates that the Elasticsearch CR uses a non-existent PVC.
        type:                  Unschedulable
 ----
 
+This status message indicates that your cluster does not have enough nodes to support your Elasticsearch redundancy policy.
```

Comment:
@mburke5678 To be more accurate, how about change `cluster` to `ES cluster`?

Reply:
@QiaolingTang Thank you for the suggestion. I made the change.